testsuite_overview.md

```diff
@@ -0,0 +1,310 @@
+Rest.li Client Test Suite
+=========================
+
+This directory contains a language-independent Rest.li client test suite.
+It is designed to be shared across languages, and provides test data and testing guidelines that can be followed when 
+testing a particular language implementation of the Rest.li client.
+
+This project also contains a Java TestNG suite that uses the test suite's data and test guidelines. 
+This example demonstrates how to use the test suite for a specific language implementation of the Rest.li client. 
+
+
+Motivation
+----
+This test suite allows us to compare implementations of the Rest.li client across
+languages.
+It will help us achieve consistently high quality implementations of the Rest.li client in multiple
+languages by:
+
+* Reducing the work required to add a new Rest.li client implementation
+* Providing a common ground where test cases can be shared amongst multiple Rest.li client implementations
+* "Certifying" Rest.li client implementations with a quantifiable measure of quality
+* Making it easier to make compatible Rest.li implementations, so a Rest.li client in one language can be used with a Rest.li server in another language. 
+
+To this effect, we've added test suite support for Python and Java, and used the test suite to create this
+[Compatibility Matrix](testsuite_compatibility_matrix.md), which compares the features supported by the implementations.
+
+Getting Started
+----
+This documentation assumes a background in Rest.li. Implementers are encouraged to explore the
```

Comment:
Put a hyperlink for Rest.li to its github page, "specifically, test developers should be familar with Rest.li data schema, IDL, and wire protocol format as demonstrated by following links:" You can move your sentence "Implementers are encouraged ...." after the bullet.